.circleci/config.yml

```diff
@@ -757,6 +757,21 @@ jobs:
             yarn workspace @businessnjgovnavigator/content build
             yarn workspace @businessnjgovnavigator/web webflow:licenses-ci-sync
 
+  sync-industries:
+    docker:
+      - image: $AWS_ACCOUNT_ID_DEV.dkr.ecr.$AWS_REGION.amazonaws.com/navigator_builder:bfs_navigator_builder-18_18_2
+        aws_auth:
+          aws_access_key_id: $AWS_ACCESS_KEY_ID_DEV
+          aws_secret_access_key: $AWS_SECRET_ACCESS_KEY_DEV
+    steps:
+      - bootstrap-code
+      - run:
+          name: Node Industry Sync
+          command: |
+            echo 'export WEBFLOW_API_TOKEN=$WEBFLOW_API_TOKEN' >> $BASH_ENV
+            yarn workspace @businessnjgovnavigator/content build
+            yarn workspace @businessnjgovnavigator/web webflow:industries-ci-sync
```

Comment:
I accidentally added this command to `package.json` awhile ago which is why that change isn't here in the diff.

There are 2 options `webflow:industries-sync` and `webflow:industries-ci-sync`.